### PR TITLE
fix raw3-1 bringup

### PR DESCRIPTION
**Pull request by benmaidel**
removed prosilica cams from raw3-1 startup

### CODE REVIEW

**Comment by floweisshardt** [comment]
CR: OK

File: cob_bringup/robots/raw3-1.xml [hunk 1]
```diff
@@ -106,17 +106,6 @@
...
-		<!-- start camera sensors -->
-		<include file="$(find cob_bringup)/drivers/prosilica.launch" >
-			<arg name="robot" value="$(arg robot)" />
-			<arg name="name" value="left" />
-		</include>
-		<include file="$(find cob_bringup)/drivers/prosilica.launch" >
-			<arg name="robot" value="$(arg robot)" />
-			<arg name="name" value="right" />
-		</include>
-	</group>
```
Comment (fmessmer) [inline cob_bringup/robots/raw3-1.xml:118]: do not remove the group tag... the xml format is broken now!

**Comment by fmessmer** [comment]
We definitely need urdf and launch checks for travis...!!!

File: cob_bringup/robots/raw3-1.xml (hunk 1 above)
Comment (benmaidel) [inline cob_bringup/robots/raw3-1.xml:118]: ups

**Comment by benmaidel** [comment]
I've readded group tag. Please review

**Comment by fmessmer** [comment]
All good now!
Created a new issue for launch file checks....#374

**Comment by NadiaHG** [comment]
CR: OK

**Pull request merged**